src/t_list.c

```diff
@@ -1001,3 +1121,69 @@ void brpoplpushCommand(client *c) {
         != C_OK) return;
     blmoveGenericCommand(c, LIST_TAIL, LIST_HEAD, timeout);
 }
+
+/* LPOP takes one key, and can return multiple elements.
+ * BLPOP takes multiple keys, but returns one element from just one key.
+ * [B]LMPOP can take multiple keys and return multiple elements from just one key.
+ *
+ * LMPOP numkeys [<key> ...] LEFT|RIGHT [COUNT count] [BLOCK timeout]
+ *
+ * [B]LMPOP
+ * 1) keyname
+ * 2) 1) element1
+ *    2) element2 */
```

Comment:
no need to document other commands, or the difference from this one. (that's for redis.io docs).
```suggestion
 * LMPOP numkeys [<key> ...] LEFT|RIGHT [COUNT count] [BLOCK timeout]
```